packages/moleculer-db-adapter-sequelize/src/index.js

```diff
@@ -58,7 +58,8 @@ class SequelizeDbAdapter {
 			this.db = new Sequelize(...this.opts);
 
 		return this.db.authenticate().then(() => {
-
+			
+			this.db.sync({alter:true});
```

Comment:
I believe this will overwrite the option set by a user though.

If I set 

```
options: {
 sync: { force: false, alter: false },
}
```

Then the alter will turn to true with the assignment on line 62 right?

Reply:
@ImTheDeveloper

you are correct .
Thanks for correcting me